<launch>

	<arg name="item_id" default="unnamed" />
	<arg name="planner_id" default="GridBased" />

  <node pkg="asb_task" exec="spraying_task_plan_maker.py" name="spraying_task_plan_maker">
    <param name="source_task_plan_file_path" value="~/w/agrosensebot_ws/src/AgroSenseBot/asb_task/config/spraying_task_plans/cornaredo.yaml" />  <!-- kind of hardcoded, but there is no way to find the source directory of a package with ros2 launch and rclpy -->
    <param name="destination_task_plan_file_path" value="~/w/agrosensebot_ws/src/AgroSenseBot/asb_task/config/spraying_task_plans/cornaredo.yaml" />  <!-- kind of hardcoded, but there is no way to find the source directory of a package with ros2 launch and rclpy -->
    <param name="frame_id" value="map" />
    <param name="dist_th" value="1.0" />
    <param name="behavior_tree_name" value="" />
    <param name="planner_id" value="$(var planner_id)" />
    <param name="task_plan_item_id" value="$(var item_id)" />
  </node>

</launch>
